### PR TITLE
Support part distribute

### PR DESCRIPTION
**Pull request by darionyaphet**

### CODE REVIEW

File: src/common/interface/meta.thrift [hunk 1]
```diff
@@ -367,7 +367,9 @@ struct StatisItem {
     3: i64                                    space_vertices,
     // The number of edges of current space
     4: i64                                    space_edges,
-    5: JobStatus                              status,
+    5: map<common.PartitionID, list<common.PartitionID>>
```
Comment (critical27) [inline src/common/interface/meta.thrift:370]: What is this part_distribute used for?

Comment (darionyaphet) [reply src/common/interface/meta.thrift:370]: try to statistical the partition correlation

**Review by panda-sheep — COMMENTED** [review]
Good job.  👍
BTW,  Which sentences will benefit from it?
In other words, in which scenarios will this be used in the future？

**Review by panda-sheep — APPROVED** [review]
Good！

**Review by bright-starry-sky — APPROVED** [review]
Nice job

**Pull request merged**